### ISSUE TITLE
web-routing-0.4.1 does not compile

### ISSUE THREAD

**Issue by peti** [philopon/web-routing#2]
Citing from http://hydra.cryp.to/build/527697/nixlog/2/raw:

```
Building web-routing-0.4.1...
Preprocessing library web-routing-0.4.1...
[1 of 4] Compiling Network.Routing.Compat ( src/Network/Routing/Compat.hs, dist/build/Network/Routing/Compat.o )
[2 of 4] Compiling Network.Routing.Dict.Internal ( src/Network/Routing/Dict/Internal.hs, dist/build/Network/Routing/Dict/Internal.o )

src/Network/Routing/Dict/Internal.hs:177:1:
    Type synonym ‘Ix'’ should have 3 arguments, but has been given 1
    In the type declaration for ‘Ix’
```

**Comment by philopon** [comment]
Thanks for your report!
I don't reproduce it in my environment, but I maybe fixed it.
Could you check fix/#2 branch?

**Comment by peti** [comment]
Yes, the `fix/#2` branch compiles fine.

**Comment by philopon** [comment]
Thanks!
I released [v0.4.2](https://hackage.haskell.org/package/web-routing-0.4.2).
